app/models/concerns/flags.rb

```diff
@@ -1,5 +1,6 @@
 module Flags
   extend ActiveSupport::Concern
+  FLAGS = %w(production, archived, alpha, beta)
 
```

Comment:
@kriskelly ok perhaps I'll remove lib/activity_flags, so we all our flag related stuff is in one place?